koku/api/migrations/0009_providerstatus_squashed_0042_auto_20200116_2048.py

```diff
@@ -244,6 +234,5 @@ class Migration(migrations.Migration):
                 null=True, on_delete=django.db.models.deletion.SET_NULL, to="api.ProviderInfrastructureMap"
             ),
         ),
-        migrations.RunPython(code=seed_cost_management_aws_account_id),
```

Comment:
this change should be reverted. never delete old migrations changes. Only add new migrations to alter the existing tables.

Reply:
@Red-HAP can you comment on how this can be done?

Reply:
reverted